lib/models/per-bundle-addon-cache/index copy.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
```

Comment:
This file is odd (`index copy.js`), maybe it is a duplicate or artifact of some refactoring?

Reply:
removed (it was a temporary copy as I was refactoring).